docs/getting_started.md

```diff
@@ -254,13 +252,20 @@ Make sure your webapp has the credentials to pull the image from the Azure Conta
 
 ![build](./images/multi-stage-webapp.png)
 
+# Customizing the Azure DevOps job container
```

Comment:
Let's remove this as a # section to keep it more of a footnote/FYI.  We'd like to keep the getting started as a learning example and then move everyone to bootstrap with the BYOC scenario.

Reply:
Addressed in comment above